content/blog/esc-automation-api-pulumi-service-provider-launch/index.md

```diff
@@ -0,0 +1,316 @@
+---
+title: "Introducing Pulumi ESC Support in Automation API and the Pulumi Service Provider"
```

Comment:
Unless I've badly missed something, we haven't yet added support for ESC to Automation API?

Reply:
Ah, nevermind--this is documenting changes that have been in for some time (~6 months). Is that right?